cluster/lib/redis/cluster/client.rb

```diff
@@ -11,6 +11,7 @@ class Client < RedisClient::Cluster
         RedisClient::Cluster::AmbiguousNodeError => Redis::Cluster::AmbiguousNodeError,
         RedisClient::Cluster::ErrorCollection => Redis::Cluster::CommandErrorCollection,
         RedisClient::Cluster::Transaction::ConsistencyError => Redis::Cluster::TransactionConsistencyError
+        RedisClient::Cluster::NodeMightBeDown => Redis::Cluster::NodeMightBeDown
```

Comment:
You'll need to declare that error in `cluster.rb`

Reply:
Added.

Reply:
Ah you have a syntax error as well. I'll cleanup your branch.

Reply:
Ah, you have push for maintainers disabled... oh well.

Reply:
Sorry about the issues with my branch and thanks for taking care of it.